### PR TITLE
Bug 1224510 - [Settings] Convert bluetooth menu to use gaia-menu

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1224510

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2d6f451ab95f0eb88f8ab4a68236491f41cef60e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=51641c886f80651f25c6bae0ad35937d393f60b9

File: apps/settings/js/panels/bluetooth/panel.js [hunk 1]
```diff
@@ -88,8 +88,7 @@ define(function(require) {
             actionMenuDialog: panel.querySelector('.paired-device-option'),
             connectOption: panel.querySelector('.connect-option'),
             disconnectOption: panel.querySelector('.disconnect-option'),
-            unpairOption: panel.querySelector('.unpair-option'),
-            cancelOption: panel.querySelector('.cancel-option')
```
Comment (gasolin) [inline apps/settings/js/panels/bluetooth/panel.js]: cancelOption is been used, please don't remove it

Comment (KevinGrandon) [inline apps/settings/js/panels/bluetooth/panel.js]: I can't find a call site for `cancelOption`, or a place where it's still accessed in code. Can you point to a line number?

Comment (gasolin) [inline apps/settings/js/panels/bluetooth/panel.js]: might be https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/panels/bluetooth/panel.js#L488 ?

Comment (KevinGrandon) [inline apps/settings/js/panels/bluetooth/panel.js]: Ah, gaia-menu should handle this now, so I'll update it. Thanks!

Comment (KevinGrandon) [inline apps/settings/js/panels/bluetooth/panel.js]: (This is removed below)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6ca6d091e021aab1bd22bf518861f9a6a15a3645

File: apps/settings/js/panels/bluetooth/panel.js (hunk 1 above)
Comment (gasolin) [inline apps/settings/js/panels/bluetooth/panel.js]: thanks!

**Pull request merged**